tests/nightly/test_large_array.py

```diff
@@ -254,8 +247,8 @@ def numpy_space_to_depth(x, blocksize):
 
 
 def test_diag():
-    h = np.random.randint(2,9)
-    w = np.random.randint(2,9)
+    h = np.random.randint(2, 9)
```

Comment:
I do not see the necessity of creating h and w variables here.

Reply:
If you decide to move forward with them, please add the with seed annotation to the test

Reply:
@apeforest Thats how the unittest for diagonal operator is implemented. 
@marcoabreu Good point will add `@with_seed()`